<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1031</width>
    <height>617</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1031</width>
    <height>617</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1031</width>
    <height>617</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>351</width>
     <height>251</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/res/chien-pao.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="skl1">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>480</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 22pt &quot;点字像素16&quot;;
</string>
   </property>
   <property name="text">
    <string>突袭</string>
   </property>
  </widget>
  <widget class="QPushButton" name="skl2">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>480</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 22pt &quot;点字像素16&quot;;</string>
   </property>
   <property name="text">
    <string>冰柱坠击</string>
   </property>
  </widget>
  <widget class="QPushButton" name="skl3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>550</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 22pt &quot;点字像素16&quot;;</string>
   </property>
   <property name="text">
    <string>圣剑</string>
   </property>
  </widget>
  <widget class="QPushButton" name="skl4">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>550</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 22pt &quot;点字像素16&quot;;</string>
   </property>
   <property name="text">
    <string>保护</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1051</width>
     <height>471</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/Desktop/image1/background.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>-10</y>
     <width>691</width>
     <height>361</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/res/ironbundle.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>460</y>
     <width>1061</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/res/skillbar.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>340</y>
     <width>361</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/res/hp.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>361</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/res/hp.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>340</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 22pt &quot;Microsoft YaHei UI&quot;;
border-image: url(:/res/chienpao.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 22pt &quot;Microsoft YaHei UI&quot;;
border-image: url(:/res/ironhands.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="msgbar">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>460</y>
     <width>991</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 26pt &quot;点字像素16&quot;;

</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>350</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="pp1">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>500</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 24pt &quot;点字像素16&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="pp2">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>500</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 24pt &quot;点字像素16&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="hp2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>20</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;点字像素16&quot;;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="hp1">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>350</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;点字像素16&quot;;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>0</y>
     <width>1031</width>
     <height>621</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/res/background.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="type">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>560</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 24pt &quot;点字像素16&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="msgbar2">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>462</y>
     <width>1031</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/res/msgbar.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>msgbar</zorder>
  <zorder>msgbar2</zorder>
  <zorder>label_11</zorder>
  <zorder>label_5</zorder>
  <zorder>label_2</zorder>
  <zorder>label</zorder>
  <zorder>skl1</zorder>
  <zorder>skl2</zorder>
  <zorder>skl3</zorder>
  <zorder>skl4</zorder>
  <zorder>label_3</zorder>
  <zorder>label_6</zorder>
  <zorder>label_7</zorder>
  <zorder>label_8</zorder>
  <zorder>label_9</zorder>
  <zorder>label_10</zorder>
  <zorder>pp1</zorder>
  <zorder>pp2</zorder>
  <zorder>hp2</zorder>
  <zorder>hp1</zorder>
  <zorder>type</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
